ups/ci_lsstcam.table

```diff
@@ -3,6 +3,25 @@
 # - Common third-party packages can be assumed to be recursively included by
 #   the "base" package.
 setupRequired(base)
+setupRequired(ap_association)
+setupRequired(ci_builder)
+setupRequired(ctrl_mpexec)
+setupRequired(daf_butler)
+setupRequired(faro)
+setupRequired(meas_algorithms)
+setupRequired(meas_astrom)
+setupRequired(meas_extensions_psfex)
+setupRequired(meas_extensions_shapeHSM)
+setupRequired(meas_extensions_photometryKron)
+setupRequired(meas_extensions_convolved)
+setupRequired(meas_modelfit)
```

Comment:
Needs multiprofit too, and probably some other packages. I'm a little surprised that some of those requirements aren't in drp_pipe already, though; I would think drp_pipe needs to setup every package required to run each pipeline with no overrides?